packages/sit-onyx/src/components/OnyxSelectInput/OnyxSelectInput.vue

```diff
@@ -142,6 +142,7 @@ defineExpose({ focus: () => input.value?.focus() });
       </div>
     </label>
 
+    <!-- TODO: #570 error message inclusion -->
```

Comment:
If we have an issue for this, we don't need a TODO here

Reply:
```suggestion
```